### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by znepb** [SwitchCraftCC/Plethora-Fabric#33]
https://plethora.madefor.cc is out-of-date, as well as very incomplete in terms of method reference. A new documentation site really should be made.

**Comment by Lemmmy** [comment]
I don't think we need to go as far as making a new documentation site, since most of the information there is still relevant, useful, and well-written. But I'm not sure what the best approach to retaining both 1.12 and 1.19 documentation is going to be. Probably a copy of the site with a version tab on the top?

The main reason this hasn't been done yet is because I never ended up porting over the docs/reference generator, since we no longer use the ASM-based method registration. That said, I do still provide (hopefully accurate) docstrings for almost all methods Plethora adds, it's just that their context is somewhat lost compared to before.

**Comment by znepb** [comment]
Maybe we could use Illuaminate, similar to CC:T? I'm not sure if it would work with Fabric, but it has worked well with CCT so it should work.
